src/js/angular/aclmanagement/controllers.js

```diff
@@ -200,4 +257,55 @@ function AclManagementCtrl($scope, toastr, AclManagementRestService, $repositori
     }, function () {
```

Comment:
`$scope.$watch` also returns a deregistration function, so it has to be added to the `subscriptions` array.